packages/babel-preset-jest/index.js

```diff
@@ -8,7 +8,11 @@
 module.exports = () => ({
   plugins: [
     require.resolve('babel-plugin-jest-hoist'),
+    // TODO: replace all the below with a single preset in Jest 26: https://github.com/facebook/jest/pull/9774
     require.resolve('@babel/plugin-syntax-object-rest-spread'),
     require.resolve('@babel/plugin-syntax-bigint'),
+    require.resolve('@babel/plugin-syntax-class-properties'),
+    require.resolve('@babel/plugin-syntax-logical-assignment-operators'),
```

Comment:
Note that this isn't supported by Node.js yet. I added it to the preset because it's already Stage 3 so probably it will be implemented "soon", but until it's implemented in Node.js the preset doesn't enable it.

Reply:
Aha! I'll back it out then 🙂